app/views/pictures/index.html.haml

```diff
@@ -0,0 +1,2 @@
+You have no pictures yet.
+%a(href='#') Add a picture
```

Comment:
Prefer the hash attributes syntax (%tag{ lang: 'en' }) over HTML attributes syntax (%tag(lang=en))